config/calibrate.py

```diff
@@ -42,28 +38,12 @@
         matchConfig.matcher.maxMatchDistArcSec = 2.0
         matchConfig.sourceSelector.active.excludePixelFlags = False
 
-# Set to match defaults currently used in HSC production runs (e.g. S15B+)
-config.catalogCalculation.plugins['base_ClassificationExtendedness'].fluxRatio = 0.95
-
-# Demand astrometry and photoCal succeed
-config.requireAstrometry = True
-config.requirePhotoCal = True
-
+# to assist with debugging
 config.doWriteMatchesDenormalized = True
 
-# Detection
-# This config matches obs_subaru, to facilitate 1:1 comparisons between DECam and HSC
-config.detection.isotropicGrow = True
-
 config.measurement.load(os.path.join(obsConfigDir, "apertures.py"))
 config.measurement.load(os.path.join(obsConfigDir, "kron.py"))
 config.measurement.load(os.path.join(obsConfigDir, "hsm.py"))
 
-# Deblender
-# These configs match obs_subaru, to facilitate 1:1 comparisons between DECam and HSC
-config.deblend.maxFootprintSize = 0
```

Comment:
With the removal of the `maxFootprintSize` default setting of 2000 in `pipe_tasks`, it appears that this will still resolve to `0`, so :+1:.